docs/PT-BR/README.md

```diff
@@ -199,10 +200,16 @@ Este método é utilizado para mover o mouse sobre o elemento assim que estiver
 
 ### `fillFieldWithText`
 
-Este médoto preenche um campo de input com um texto assim que estiver visível.
+Este método preenche um campo de input com um texto assim que estiver visível.
 
 [Exemplo](./EXAMPLES.md#fillFieldWithText)
 
+### `fillFieldWithTextAndPressEnter`
```

Comment:
This part already exists later on in this file. Maybe, move it from there to here.
By the way, the description of the existing one is more informative.